src/lock_file/update.rs

```diff
@@ -1099,14 +1099,18 @@ fn make_unsupported_pypi_platform_error(
     }
 
     // Find all the features that are excluding the current platform.
-    let features_without_platform = grouped_environment.features().filter_map(|feature| {
-        let platforms = feature.platforms.as_ref()?;
-        if !platforms.value.contains(&current_platform) {
-            Some((feature, platforms))
-        } else {
-            None
-        }
-    });
+    let features_without_platform = grouped_environment
+        .environments()
+        // The default feature is included if its platforms are included in the environment
+        .flat_map(|e| e.features(e.manifest().from_default_feature.platforms))
```

Comment:
Is this the correct flag to include/exclude the default feature here? I am really not sure...